Probesys/bileto
# imp: Simplify the answer form

## Related issue(s)



https://github.com/Probesys/bileto/issues/35

## Changes



- remove the "is solution" checkbox
- mark a message as solution if the status is set to "resolved"
- move the status select field in a popover next to the answer button
- remove the form-new-answer Stimulus controller as it was now useless
- add a button-select Stimulus controller to update the content of a button based on a selected radio button

## How to test manually



- create a ticket and mark your user as requester
- assign the ticket to another user
- answer to the ticket as requester → check you can't select the status and that it stays as "new"
- answer to the ticket as assignee → check that the status passes to "in progress"
- answer to the ticket as assignee and select "Answer and resolve" → check that the status passes to "resolved" and that your message is marked as solution
- answer to the ticket as requester → check that the status passes to "in progress"

## Checklist
